README.md

```diff
@@ -955,6 +955,7 @@ See also <https://en.wikipedia.org/wiki/List_of_streaming_media_systems>, <https
 - [gonic](https://github.com/sentriz/gonic) - Lightweight music streaming server. Subsonic compatible. `GPL-3.0` `Go`
 - [Groove Basin](https://github.com/andrewrk/groovebasin) - Music player server with a web-based user interface inspired by Amarok 1.4. `MIT` `Nodejs`
 - [koel](http://koel.phanan.net/) - Personal music streaming server that works. ([Source Code](https://github.com/phanan/koel)) `MIT` `PHP`
+- [KooZic](https://koozic.net/) - Music server with powerful playlist features. Subsonic compatible ([Demo](https://demo.koozic.net/public), [Source Code](https://github.com/DocMarty84/koozic)) `LGPL-3.0/MIT` `Python`
```

Comment:
My apologies for missing this earlier, but your missing a `.` at the end of the description. 
Could you update the description to read something along the lines of "Music server with powerful playlist features and subsonic compatibility.

Reply:
Done :smile: